src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultEditorTemplates.cs

```diff
@@ -288,18 +281,26 @@ public static IHtmlContent ObjectTemplate(IHtmlHelper htmlHelper)
                     readOnly: false,
                     additionalViewData: null);
 
-                content.Append(templateBuilder.Build());
-
                 if (!propertyMetadata.HideSurroundingHtml)
                 {
-                    content.Append(" ");
-                    content.Append(htmlHelper.ValidationMessage(
+                    var divTag = new TagBuilder("div");
+                    divTag.AddCssClass("editor-field");
+
+                    var innerContent = new BufferedHtmlContent();
```

Comment:
Oh, I see. Can't put the same `TagBuilder` into the final content. Might do with a couple of helper methods to restore alignment between code and generated HTML.